test/commands/scanner/rule/describe.test.ts

```diff
@@ -1,101 +1,90 @@
+				expect(ruleJson.message).to.equal('Too many fields', 'Rule message should match');
+			});
 		});
 
 		describe('Test Case: Multiple matching rules', () => {
 			// Both tests will test for the presence of this warning string in the output, so we might as well format it up here.
 			const formattedWarning = messages.getMessage('output.multipleMatchingRules', ['3', 'constructor-super']);
 
-			setupCommandTest
-				.command(['scanner:rule:describe', '--rulename', 'constructor-super'])
-				.it('Displayed output matches expectations', ctx => {
-					// First, verify that the warning was printed at the start like it should have been.
-					expect(ctx.stderr).to.contain('WARNING: ' + formattedWarning, 'Warning message should be formatted correctly');
+			it('Displayed output matches expectations', () => {
+				const output = runCommand(`scanner rule describe --rulename constructor-super`);
+				const ctx = output.shellOutput;
+				// First, verify that the warning was printed at the start like it should have been.
+				expect(ctx.stderr).to.contain('WARNING: ' + formattedWarning, 'Warning message should be formatted correctly');
 
-					// Next, verify that there are rule descriptions that are distinctly identified.
-					const regex = /=== Rule #1\n\nname:\s+constructor-super(.*\n)*=== Rule #2\n\nname:\s+constructor-super(.*\n)*=== Rule #3\n\nname:\s+constructor-super/g;
-					expect(ctx.stdout).to.match(regex, 'Output should contain three rules named constructor-super for each eslint based engine');
-				});
+				// Next, verify that there are rule descriptions that are distinctly identified.
+				const regex = /=== Rule #1\n\nname:\s+constructor-super(.*\n)*=== Rule #2\n\nname:\s+constructor-super(.*\n)*=== Rule #3\n\nname:\s+constructor-super/g;
+				expect(ctx.stdout).to.match(regex, 'Output should contain three rules named constructor-super for each eslint based engine');
+			});
 
-			// TODO: THIS TEST IS FAILING BECAUSE THE WARNING FOR DEFINITELYFAKERULE IS BEING INCLUDED IN THE WARNINGS HERE.
-			//  WHEN THE CAUSE FOR THAT IS DETERMINED AND RESOLVED, RE-ENABLE THIS TEST.
-			/*
-			describeTest
-			  .stdout()
-			  .stderr()
-			  .command(['scanner:rule:describe', '--rulename', 'WhileLoopsMustUseBraces', '--json'])
-			  .it('--json flag yields correct results', ctx => {
-				const ctxJson = JSON.parse(ctx.stdout);
+			it('--json flag yields correct results', () => {
+				const output = runCommand(`scanner rule describe --rulename constructor-super --json`);
+				const ctxJson = output.jsonOutput;
 
 				// Verify we got the right number of rules.
-				expect(ctxJson.result).to.have.lengthOf(2, 'Should be two included rules');
-				// Verify that there's only one warning.
-				expect(ctxJson.warnings).to.have.lengthOf(1, 'Should be only one warning, instead ' + ctx.stdout);
-				// Verify that the warning looks right.
-				expect(ctxJson.warnings[0]).to.equal(formattedWarning, 'Warning should match');
-			  });
-			 */
```

Comment:
Enabled this test but changed the rule to `constructor-super` like the other test in this suite. It passes now. Hooray.